Gemfile

```diff
@@ -33,10 +33,10 @@ gem 'druid-tools'
 gem 'moab-versioning', '~> 2.0'
 
 # Database
-group :integration, :staging, :production do
```

Comment:
Can you say more about removing these groups? FWIW, it looks like `staging` is a legitimate environment for this application, judging by https://github.com/sul-dlss/sdr-services-app/blob/master/config/deploy/stage.rb and https://github.com/sul-dlss/shared_configs/blob/sdr-services-app_stage/config/environments/staging.rb.

I also see `:integration` in `config/database.rb`, but I can't vouch for it otherwise.

Reply:
The staging environment will still use this group because gem groups and environments are different things.

Reply:
I get that gem groups and environments are different things. What is not clear to me is what gem groups a novel Rails environment (like `:staging`) will use.

Reply:
Looks good now.